cumulusci/core/config/SfdxOrgConfig.py

```diff
@@ -74,6 +76,11 @@ def sfdx_info(self):
         )
         return sfdx_info
 
+    @property
+    def sfdx(self):
+        """This exists primarily so that I can mock it out for testing"""
+        return sfdx
```

Comment:
I'd rather rename the module to sfdx_org_config.py so that we can patch it without needing this. This module was only created one or two releases ago so I'm not worried about backwards-compatibility.

Reply:
How does renaming the module make patching easier? I feel like I'm missing something that should be obvious.

Reply:
Right now we can't do `mock.patch("cumulusci.core.config.SfdxOrgConfig.sfdx")` because cumulusci/core/config/__init__.py imports the SfdxOrgConfig class, so cumulusci.core.config.SfdxOrgConfig ends up referring to the class and there is no way to target the module. If we rename the module then we can patch `cumulusci.core.config.sfdx_org_config.sfdx` and be sure we're targeting the module rather than the class.